package.json

```diff
@@ -137,7 +137,7 @@
     "tsconfig-paths-webpack-plugin": "^3.2.0",
     "typedoc": "^0.15.6",
     "typescript": "^3.9.6",
-    "webpack": "^4.35.0",
+    "webpack": "^4.46.0",
```

Comment:
Is there a reason why this needed to be updated as part of this PR? Is there a feature that we needed a new version for? Not a problem to update it, but this is the kind of change would be better done in its own commit, with a message explaining why it was updated, in the (admittedly unlikely) chance that we need to `bisect` and find a breaking change elsewhere in the project.